src/ansible_compat/runtime.py

```diff
@@ -199,6 +203,31 @@ def install_collection(
             _logger.error(msg)
             raise InvalidPrerequisiteError(msg)
 
+    def install_collection_from_disk(
+        self, path: str, destination: Optional[Union[str, pathlib.Path]] = None
+    ) -> None:
+        """Build and install collection from a given disk path."""
+        if self.version_in_range(upper="2.11"):
```

Comment:
Reverse this
```suggestion
        if not self.version_in_range(upper="2.11"):
            return self.install_collection(path, destination=destination)
```
and the rest of the code could be dedented.